apps/portal/src/content/docs/components/toggle-group.mdx

```diff
@@ -0,0 +1,215 @@
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the root element",
+      required: false,
+      value: "string",
+    },
+    {
+      name: "variant",
+      description: "Variant of the toggle group",
+      required: false,
+      value: "'default' | 'outline' | 'compact'",
+    },
+    {
+      name: "type",
+      description: "Type of the toggle group",
+      required: true,
+      value: "'single' | 'multiple'",
+    },
+    {
+      name: "value",
+      description:
+        "Controlled stateful value of the item(s) that is/are pressed",
+      required: false,
+      value: "string | string[]",
+    },
+    {
+      name: "defaultValue",
+      description:
+        "Value of the item(s) that is/are pressed when initially rendered",
+      required: false,
+      value: "string | string[]",
+    },
+    {
+      name: "onValueChange",
+      description:
+        "Callback that fires when the value of the toggle group changes",
+      required: false,
+      value: "(value: string | string[]) => void",
+    },
+    {
+      name: "disabled",
+      description: "Whether the group is disabled from user interaction",
+      required: false,
+      value: "boolean",
+      defaultValue: "false",
+    },
+    {
+      name: "rovingFocus",
+      description:
+        "Whether the group should maintain roving focus of its buttons",
+      required: false,
+      value: "boolean",
+      defaultValue: "true",
+    },
+    {
+      name: "loop",
+      description: "Whether the focus should loop around when reaching the end",
+      required: false,
+      value: "boolean",
+    },
+    {
+      name: "orientation",
+      description: "Orientation of the toggle group",
+      required: false,
+      value: "'horizontal' | 'vertical'",
+    },
+    {
+      name: "dir",
+      description: "Text direction of the toggle group",
+      required: false,
+      value: "'ltr' | 'rtl'",
+    },
+  ]}
+/>
+
+### Item
+
+The `Item` component is used to define each toggle button within the group.
+
+```typescript jsx
+<ToggleGroup.Item value="1">Option 1</ToggleGroup.Item>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "children",
+      description: "Content to display",
+      required: true,
+      value: "ReactNode",
+    },
+    {
+      name: "className",
+      description: "Additional class names to apply to the item",
+      required: false,
+      value: "string",
+    },
+    {
+      name: "value",
+      description: "Value of the toggle item",
+      required: true,
+      value: "string",
+    },
+    {
+      name: "variant",
+      description: "Variant of the toggle item",
+      required: false,
+      value: "'default' | 'outline' | 'compact'",
+    },
+    {
+      name: "size",
+      description: "Size of the toggle item",
+      required: false,
+      value: "'small' | 'medium' | 'large'",
```

Comment:
What are the defaults for these? Might be worth adding to make sure people don't specify when they want the same value as the default